presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -39,6 +41,8 @@
     public static final String PARTITIONED_BY_PROPERTY = "partitioned_by";
     public static final String BUCKETED_BY_PROPERTY = "bucketed_by";
     public static final String BUCKET_COUNT_PROPERTY = "bucket_count";
+    public static final String FIELD_DELIM_PROPERTY = "field_delim";
```

Comment:
I'm thinking we could call these `fields_delimited_by` and `lines_terminated_by`